src/components/Header/Header.tsx

```diff
@@ -24,9 +24,15 @@ const Header = React.forwardRef<HTMLDivElement, IHeader>((props, ref) => {
     >
       <div className="flex gap-1 items-center">
         {startAdornment}
-        <Link className="font-medium text-white" to="/">
-          {title}
-        </Link>
+        {title === 'SOS Rio Grande do Sul' ? (
+          <Link className="font-medium text-white" to="/">
+            {title}
+          </Link>
+        ) : (
+          <span className="font-medium text-white">
+            {title}
+          </span>
```

Comment:
Será que conseguimos manter o comportamento de voltar, já presente na setinha/chevron, também no título?

Reply:
eu tentei essa possibilidade, mas quando o EditShelterSupply enviar os parâmetros para o header

`<Header
          title="Editar Itens"
          className="bg-white [&_*]:text-zinc-800 border-b-[1px] border-b-border"
          startAdornment={
            <Button
              variant="ghost"
              className="[&_svg]:stroke-blue-500"
              onClick={() => navigate(`/abrigo/${shelterId}`)}
            >
              <ChevronLeft size={20} />
            </Button>
          }
/>`

neste momento, eu não achei como poderia enviar/receber o navigate como é feito com o button para o header e mudar a rota no link

então para não arrumar uma coisa e estragar a outra achei melhor desta forma, ate porque tenho pouca experiência em projeto tão extenso

Reply:
> Será que conseguimos manter o comportamento de voltar, já presente na setinha/chevron, também no título?

Atualize a interface **IHeader** para incluir a propriedade **to**:
```
export interface IHeader extends React.ComponentPropsWithoutRef<'div'> {
  title: string;
  startAdornment?: React.ReactNode;
  endAdornment?: React.ReactNode;
  to?: string;
}
```
Modifique o componente **Header** para renderizar condicionalmente o título como um link se a propriedade **to** for fornecida:
```
const Header = React.forwardRef<HTMLDivElement, IHeader>((props, ref) => {
  const {
    endAdornment,
    startAdornment,
    title,
    className = '',
    to,
    ...rest
  } = props;
```
```
{to ? (
   <Link className="font-medium text-white" to={to}>
     {title}
   </Link>
) : (
   <span className="font-medium text-white">{title}</span>
)} 
```

Certifique-se de refatorar outros arquivos que usam o componente **Header** para passar a propriedade **to** conforme necessário.

Reply:
sua explicação me ajudou muito a melhorar o código, obrigado!